server/channels/app/platform/web_conn.go

```diff
@@ -88,6 +89,7 @@ type WebConn struct {
 	Locale           string
 	Sequence         int64
 	UserId           string
+	OriginClient     string
```

Comment:
Does this need to be exported?

Reply:
Not really, we are using the config to get it there and then it's used internally. I can update :+1: